ap_src/templates/api_pages/team_calendar.html

```diff
@@ -104,21 +105,21 @@ <h1 class="title is-4">{{ team.name }}
                         {% check_absences recurring_absence_dates|get_key:user.user.username year month_num day as check_recurring_absence %}
                         {% check_half_day half_days_data|get_key:user.user.username year month_num day as check_half_day %}
                         {% if check_recurring_absence %}
-                            <td style="background-color: #ffaa00",  id="{{user.user.username}}/{{year}}-{{month_num}}-{{day}}/TRUE" data-absence-type="R"></td>
+                            <td data-editable="{{editable}}" style="background-color: #ffaa00",  id="{{user.user.username}}/{{year}}-{{month_num}}-{{day}}/TRUE" data-absence-type="R"></td>
```

Comment:
1. There are two spaces before id, delete one of them.
2. Delete the comma after style="background-color: #ffaa00"